mongodb/driver/manager/construct.xml

```diff
@@ -85,27 +85,27 @@ $manager = new MongoDB\Driver\Manager("mongodb://localhost:27017");
 ?>
 ]]>
    </programlisting>
-   <para>Connecting to a Replica Set</para>
+   <para>Connecting to a replica set</para>
    <programlisting role="php">
 <![CDATA[
 <?php
 
-$manager = new MongoDB\Driver\Manager("mongodb://localhost:27017/?replicaSet=my-replica");
+$manager = new MongoDB\Driver\Manager("mongodb://rs1.example.com,rs2.example.com/?replicaSet=myReplicaSet");
 
 ?>
 ]]>
    </programlisting>
-   <para>Connecting to a sharded MongoDB (e.g. mongos)</para>
+   <para>Connecting to a shard cluster (i.e. one or more mongos instances)</para>
    <programlisting role="php">
 <![CDATA[
 <?php
 
-$manager = new MongoDB\Driver\Manager("mongodb://localhost:27018/");
+$manager = new MongoDB\Driver\Manager("mongodb://mongos1.example.com,mongos2.example.com/");
 
 ?>
 ]]>
    </programlisting>
-   <para>Connecting to MongoDB using username &amp; password</para>
+   <para>Connecting to MongoDB with authentication credentials for a particular user and database.</para>
    <programlisting role="php">
```

Comment:
I liked the [standalone|mongosX|rsX].example.com naming convention.
It also makes the example clear without reading the surroundings.